packages/bootstrap-4/src/TextareaWidget/TextareaWidget.tsx

```diff
@@ -26,30 +26,44 @@ const TextareaWidget = ({
 }: CustomWidgetProps) => {
   const _onChange = ({
     target: { value },
-  }: React.ChangeEvent<HTMLInputElement>) =>
+  }: React.ChangeEvent<HTMLTextAreaElement>) =>
     onChange(value === "" ? options.emptyValue : value);
-  const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement>) =>
-    onBlur(id, value);
+  const _onBlur = ({
+    target: { value },
+  }: React.FocusEvent<HTMLTextAreaElement>) => onBlur(id, value);
   const _onFocus = ({
     target: { value },
-  }: React.FocusEvent<HTMLInputElement>) => onFocus(id, value);
+  }: React.FocusEvent<HTMLTextAreaElement>) => onFocus(id, value);
 
   return (
-    <TextField
-      id={id}
-      label={label || schema.title}
-      placeholder={placeholder}
-      disabled={disabled || readonly}
-      value={value}
-      required={required}
-      autoFocus={autofocus}
-      multiline={true}
-      rows={options.rows || 5}
-      error={rawErrors.length > 0}
-      onChange={_onChange}
-      onBlur={_onBlur}
-      onFocus={_onFocus}
-    />
+    <>
+      <label htmlFor={id}>
+        {label || schema.title}
+        {required && (
+          <span
+            aria-hidden
+            className={rawErrors.length > 0 ? "text-danger" : ""}>
+            {" "}
```

Comment:
Could this spacing be done with css, like `ml-1` or something?

Reply:
sure 👍